client/volume_create.go

```diff
@@ -12,6 +14,9 @@ import (
 func (cli *Client) VolumeCreate(ctx context.Context, options volumetypes.VolumesCreateBody) (types.Volume, error) {
 	var volume types.Volume
 	resp, err := cli.post(ctx, "/volumes/create", nil, options, nil)
+	if resp.statusCode == http.StatusNotModified {
+		return volume, fmt.Errorf("A volume named %s has already been created.", options.Name)
```

Comment:
I wonder if we should only return an error if options are different (which is something we track now).
This is a breaking CLI change as is.